bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
```

Comment:
Do you mean `import http from 'http'`? I don't think that works until you're in a file that was imported. I guess I could make a file that just does `require('babel-register');require('./proxy.js')` and call that instead though.